src/pipelines/epidemiology/gb_authority.py

```diff
@@ -211,7 +211,18 @@ def parse(self, sources: Dict[str, str], aux: Dict[str, DataFrame], **parse_opts
 
         # For consistency across countries, do not report confirmed / deceased counts since we get
         # that data from WHO / ECDC
-        data = data[["key", "date", "new_tested", "total_tested"]]
+        data = data[
+            [
+                "key",
+                "date",
+                "new_confirmed",
+                "total_confirmed",
+                "new_deceased",
+                "total_deceased",
+                "new_tested",
+                "total_tested",
+            ]
+        ]
```

Comment:
So we have confirmed that the data looks OK now? If so, let's remove this whole chunk of code since it's outdated.

Reply:
It depends on what you mean by OK. I've confirmed that the missing data is no longer missing.